APIScrape/scrape.php

```diff
@@ -20,10 +20,11 @@ function createDatabase(\PDO $db) {
                 CREATE TABLE `breed` (
                 `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
                 `breed` varchar(255) NOT NULL DEFAULT '',
+                `favourite_cat` int(11) unsigned DEFAULT NULL,
```

Comment:
fav_id . instead of favourite_cat

Reply:
Agreed - we know it's a cat, what this column actually is is the ID of the cat that is the favourite for that breed.